modules/grpc/metadata.json

```diff
@@ -8,7 +8,8 @@
         "1.41.0",
         "1.47.0",
         "1.48.1",
-        "1.48.1.bcr.1"
+        "1.48.1.bcr.1",
+	"1.48.1.bcr.2"
```

Comment:
nit - could you adjust this to use spaces?

Reply:
Done.